docs/source/tutorial/package_a_project.md

```diff
@@ -6,23 +6,23 @@ Kedro also has an advanced feature which supports packaging on a pipeline level
 
 ## Add documentation to your project
 
-Kedro uses the [Sphinx framework](https://www.sphinx-doc.org) to build any documentation that you create about your project.
+There are several documentation frameworks for Python projects, with [Sphinx](https://www.sphinx-doc.org) being among the most popular ones. You can leverage Sphinx to build the documentation of your Kedro project.
 
-To install a compatible version of Sphinx, run the following from the project root directory:
+To install Sphinx, run the following:
```

Comment:
Not even mentioning that a newer version could be used, we could save that for Kedro 0.19